index.js

```diff
@@ -2,18 +2,18 @@ import { ethers, Wallet } from 'ethers'
 import fetch from 'node-fetch'
 import { publishCast } from '@standard-crypto/farcaster-js'
 import { createClient } from '@supabase/supabase-js'
-
+import * as dotenv from 'dotenv'
+dotenv.config()
 /*
  * Constants
  */
 const SUPABASE_URL = 'https://kpwbglpxjuhiqtgtvenz.supabase.co'
 const READ_ONLY_SUPABASE_KEY =
   'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imtwd2JnbHB4anVoaXF0Z3R2ZW56Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2NTgzNzg2MjEsImV4cCI6MTk3Mzk1NDYyMX0.zecokpSRK0MI_nOaSAgFZJCMkPSpEXraPKqQD5fogE4'
 const supabase = createClient(SUPABASE_URL, READ_ONLY_SUPABASE_KEY)
-
 const NCBOT_USERNAME = 'ncbot'
-const NCBOT_ADDRESS = '0xb79aF3B13F54c0739F0183f1207F7AB80EDd40DE'
 const RECAST_PREFIX = 'recast:farcaster://casts/'
+const NCBOT_FID = 258
```

Comment:
ncbot ID is 1026

Reply:
that was my bad... I pulled yours by mistake.

Reply:
done